package com.github.savitoh.centralerroapi.exception;

public class RecuperaUsuarioException extends RuntimeException{

    public RecuperaUsuarioException(String message) {
        super(message);
    }
}
